packages/components/src/FilterBar/_docs/FilterBar.mdx

```diff
@@ -118,44 +164,28 @@ unless there is a default value provided, of which it will be in the active filt
 
 When removed from the active filters, the filter will be added to the `Add Filter` menu options.
 
-#### Usage
+##### `isUsableWhen?: (state: FiltersState<ValuesMap>) => boolean`
 
-We also provide a `Filters<ValuesMap>` type you can use for improved type safety.
+Note: _Usable_ refers to a filter showing in either the Active Filters or as an option in the Add Filters menu.
 
-```tsx
-import { FilterBar, Filters, DateRange, FilterMultiSelect } from "@kaizen/components"
+Use this attribute to apply a condition to a filter based on the state of another filter ([example here](#dependent-filters)).
 
-type ValuesMap = {
-  filterId1: string
-  filterId2: string[]
-  filterId3: DateRange
-}
-
-const filters = [
-  {
-    id: "filterId1",
-    name: "Filter 1",
-    Component: <FilterBar.Select {...props} />,
-  },
-  {
-    id: "filterId2",
-    name: "Filter 2",
-    Component: (
-      <FilterBar.MultiSelect {...props}>
-        {/* Compose your structure using subcomponents of FilterMultiSelect */}
-        {() => <FilterMultiSelect.ListBox {...props} />}
-      </FilterBar.MultiSelect>
-    ),
-  },
-  {
-    id: "filterId3",
-    name: "Filter 3",
-    Component: <FilterBar.DateRangePicker {...props} />,
-    isRemovable: true,
-  },
-] satisfies Filters<ValuesMap>
+The `state` arg will provide you with the following attributes for all the filters (mapped to the keys of the `ValuesMap`):
+```ts
+id: keyof ValuesMap
+name: string
+isRemovable?: boolean
+isOpen: boolean
+value?: ValueMap[keyof ValuesMap]
+isActive: boolean
```

Comment:
Writing this out really made me wonder whether a consumer would ever need all these attributes when writing a condition. I can only reasonably think of `isActive` and `value` making sense to use.

Should I revise the types?

Reply:
Agreed, probably not `isOpen` and `isRemovable`. Maybe the `id` and `name` could be useful if they want to trigger analytics/other funcs that might string match

Reply:
Oh true. Didn't think about potentially using it for analytics. Although the condition gets run for every filter on every state update, so I don't know if they'd actually want to do that 😂 Their choice though.